domains.js

```diff
@@ -0,0 +1,21 @@
+export default [
```

Comment:
some registrars that are listed are missing here.

Reply:
Sorry! While I was working on this PR i guess you guys added new domains. Commit incoming :rocket: